test/merger/test_general_merger.py

```diff
@@ -5,11 +5,16 @@
 from opensquirrel import Circuit, CircuitBuilder
 from opensquirrel.default_instructions import Ry, Rz
 from opensquirrel.ir import BlochSphereRotation, Float, Qubit
-from opensquirrel.passes.merger import general_merger
-from opensquirrel.passes.merger.general_merger import compose_bloch_sphere_rotations, rearrange_barriers
+from opensquirrel.passes.merger import SingleQubitGatesMerger
+from opensquirrel.passes.merger.general_merger import Merger, compose_bloch_sphere_rotations, rearrange_barriers
 from test.ir_equality_test_base import modify_circuit_and_check
 
 
+@pytest.fixture(name="merger")
+def merger() -> type[SingleQubitGatesMerger]:
+    return SingleQubitGatesMerger
+
+
```

Comment:
Remove all tests that are in `test_single_qubit_gate_merger.py` from `test_general_merger.py` (and also this fixture is then no longer needed as well.

Reply:
Sorry, forgot to remove the duplicated tests. I've committed your suggestions and processed your feedback.